frontend/src/pages/CourseSelector/courseDescription/courseDescription.less

```diff
@@ -90,6 +90,20 @@
   gap: 20px;
 }
 
+.progressBarWrapper {
+  width: '100%';
+  background-color: 'whitesmoke';
+  border-radius: 40;
+}
 
+.progressBar {
+  height: '100%';
+  border-radius:40;
+  text-align: 'right'
+}
 
-
+.progressText {
+  padding: 10;
```

Comment:
For `padding`, could you specify the units being used i.e. `px`? And also, for `color`, replace the value to its hex code i.e. `#000`